astropy/utils/iers/tests/test_leap_second.py

```diff
@@ -80,7 +79,7 @@ def test_open_leap_seconds_list(self, file):
     def test_open_system_file(self):
         ls = iers.LeapSeconds.open(SYSTEM_FILE)
         expired = ls.expires < Time.now()
-        if ON_CI and expired:
+        if expired:
             pytest.skip("CI system leap second file is expired.")
```

Comment:
Delete "CI" in the message too, I guess (below as well)

Reply:
Done. Thanks for the review!